io.github.mpc_qt.mpc-qt.yml

```diff
@@ -0,0 +1,294 @@
+app-id: io.github.mpc_qt.mpc-qt
+runtime: org.kde.Platform
+runtime-version: '6.7'
```

Comment:
current is 6.8

Reply:
With 6.8, mpv didn't build, with the same error as in https://github.com/terrapkg/packages/issues/3338.

Reply:
then you need to figure out why. 6.7 will be EOL soon enough.

Reply:
I was hoping that error (`x86_64-unknown-linux-gnu/bin/ld: /app/lib/libmpv.so: undefined reference to av_dovi_find_level@LIBAVUTIL_59`) will get fixed some time in the future.

Reply:
that looks like it's linking against the wrong version. There is a ffmpeg in the runtime too. 

it's not the runtime's fault.

Reply:
Downgrading ffmpeg to 7.0.2 (which includes libavutil 59.8.100 instead of 59.39.100) fixes it.
Hopefully the org.kde.Platform 6.9 will work with ffmpeg 7.1.
h265 decoding (and hardware h264 decoding) may even [work out of the box](https://invent.kde.org/packaging/flatpak-kde-runtime/-/issues/67) with libmpv.

Reply:
Well, it fixes the build, but then h265 decoding (and hardware h264 decoding) don't work either. So I'm guessing org.freedesktop.Platform.ffmpeg-full 24.08 uses ffmpeg 7.0.2.
Hopefully, 25.08 will use ffmpeg 7.1 and work with libmpv.

Reply:
> Downgrading ffmpeg to 7.0.2 (which includes libavutil 59.8.100 instead of 59.39.100) fixes it.

6.8 uses that exact same version 7.0.2